### DISCUSSION TITLE
help for launcher

### DISCUSSION THREAD

**Discussion by Unixware** [i3/i3#6219] [Q&A] [answered]
hello!

I found an example of custom launcher on the https://i3wm.org/docs/userguide.html site, ($mode_launcher )  (para 4.6) that config works ok for me but it stays enabled till I press ESC or Return, any ideas if is possible to have a such launcher that goes out when I make my choice? TIA

**Comment by yusufaktepe** [comment, marked as answer]
Append `, mode "default"` to your bindings.
```i3
...
mode "$mode_launcher" {
    bindsym f exec firefox, mode "default"
    bindsym t exec thunderbird, mode "default"

    bindsym Escape mode "default"
    bindsym Return mode "default"
}
...
```

**Reply by Unixware** [reply]
yay! THANKS!!!!!!!!
